vmdb/app/models/miq_server/server_smart_proxy.rb

```diff
@@ -116,7 +116,7 @@ def scan_sync_vm(ost)
         temp_args['ems'][:use_vim_broker]      = MiqServer.use_broker_for_embedded_proxy?(temp_args['ems']['connect_to'])
         temp_args['ems'][:vim_broker_drb_port] = MiqVimBrokerWorker.drb_port if temp_args['ems'][:use_vim_broker]
         # RHEV-M setup
-        if v.vendor.to_s == 'RedHat'
+        if v.vendor.to_s == 'Red Hat'
```

Comment:
I can take a look.  I'm fine leaving this open for a bit while we fix some real problems related to this code.  I can't see a rush around this...